docs/CMakeLists.txt

```diff
@@ -17,8 +17,10 @@ if(NOT ENABLE_DOCKER)
         set(InferenceEngine_DIR ${CMAKE_BINARY_DIR})
     endif()
 
-    add_subdirectory(onnx_custom_op)
-    add_subdirectory(template_extension)
+    if (NGRAPH_ONNX_IMPORT_ENABLE)
+        add_subdirectory(onnx_custom_op)
+        add_subdirectory(template_extension)
```

Comment:
why template extension requires ONNX Importer?

Reply:
template extension was extended to to use onnx_importer's register_operator function. It's the essence of this PR - to demostrate the end user how custom operator can be used in ONNX models.

Reply:
Because @mateusztabaka in this PR added an example, how user can map custom ONNX operations.

This approach requires ONNX importer library.

Reply:
> template extension was extended to to use onnx_importer's register_operator function

It should be done conditionally if ONNX importer is enabled. No needs to disable the whole `template_extensions` library build.